src/cider/exec/nextgen/context/StringHeap.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef NEXTGEN_CONTEXT_STRING_HEAP_H
+#define NEXTGEN_CONTEXT_STRING_HEAP_H
+
+#include "cider/CiderAllocator.h"
```

Comment:
Minor: Hmm, better to put under https://github.com/intel/BDTK/tree/main/src/cider/type/data/string?

Reply:
I think it may bring extra link issue. and files under this folder will be removed in the future.